### PR TITLE
Styling tweaks following BS3 upgrade.

### PR DESCRIPTION
**Pull request by manstis**
The Guided Decision Table Editor Wizard was somewhat broken.

### CODE REVIEW

**Comment by manstis** [comment]
Hi @cristianonicolai, could you please merge this PR?

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/FactPatternConstraintsPageViewImpl.ui.xml [hunk 1]
```diff
@@ -41,8 +44,8 @@
           <gwt:ScrollPanel ui:field="availableFieldsContainer" styleName="{res.css.wizardDTableList}" height="220px" width="180px"/>
         </gwt:VerticalPanel>
         <gwt:VerticalPanel ui:field="buttonBar" styleName="{res.css.wizardDTableButtons}">
-          <b:Button ui:field="btnAdd" enabled="false">&gt;&gt;</b:Button>
-          <b:Button ui:field="btnRemove" enabled="false">&lt;&lt;</b:Button>
+          <b:Button ui:field="btnAdd" enabled="false" icon="ANGLE_RIGHT" width="50px"/>
```
Comment (cristianonicolai) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/FactPatternConstraintsPageViewImpl.ui.xml]: Instead of specifying width, you could use iconFixedWidth=true, which adds the default width for icons.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml [hunk 2]
```diff
@@ -63,36 +66,36 @@
         </gwt:VerticalPanel>
       </gwt:HorizontalPanel>
       <gwt:VerticalPanel ui:field="patternDefinition" visible="false">
-        <gwt:HorizontalPanel ui:field="bindingContainer" styleName="{res.css.wizardDTableFieldContainerValid} form-group">
-          <b:Label text="{i18n.Binding}:" styleName="{res.css.wizardDTableFields} help-block"/>
-          <guvnor:BindingTextBox ui:field="txtBinding" enabled="false" addStyleNames="{res.css.wizardDTableFields}"></guvnor:BindingTextBox>
+        <gwt:HorizontalPanel ui:field="bindingContainer" styleName="{res.css.wizardDTableFieldContainerValid}">
+          <b:Label text="{i18n.Binding}:" styleName="{res.css.wizardDTableFields}"/>
```
Comment (cristianonicolai) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml]: Have a look if you could reuse org.uberfire.client.views.pfly.widgets.FormLabelHelp or org.uberfire.client.views.pfly.widgets.HelpIcon in here. Another point is that you probably want to use FormLabel instead of Label.

**Comment by cristianonicolai** [comment]
@manstis I noticed that in some cases you're using setting styleName instead of addStyleNames. I think is best to keep the default styling and only overwrite what is necessary using a new css class instead of clearing the default style. What do you think?

**Comment by manstis** [comment]
Thanks for the great review! Let me try again later today :-)

**Comment by manstis** [comment]
Closing this PR as I'll submit a new one from my fork...
